assets/src/design-system/components/contextMenu/menuItem.js

```diff
@@ -68,6 +68,7 @@ export const MenuItem = ({
   onFocus,
   shortcut,
   Icon,
+  ...menuItemProps
```

Comment:
Allows us to pass `onMouseDown` to the menu items - this enables us to keep the element focused while clicking the menu using `ev.preventDefault`.